var/spack/repos/builtin/packages/py-cdo/package.py

```diff
@@ -12,10 +12,21 @@ class PyCdo(PythonPackage):
 
     pypi = "cdo/cdo-1.3.2.tar.gz"
 
-    version("1.3.2", sha256="9f78879d90d14134f2320565016d0d371b7dfe7ec71110fd313868ec1db34aee")
+    maintainers = ["Try2Code", "skosukhin"]
 
-    depends_on("cdo")
+    version("1.5.6", sha256="fec1a75382f01b3c9c368e8f143d98b12323e06975663f87d9b60c739ae1d335")
+    version(
+        "1.3.2",
+        sha256="9f78879d90d14134f2320565016d0d371b7dfe7ec71110fd313868ec1db34aee",
+        deprecated=True,
+    )
 
```

Comment:
```python
depends_on("python@2.7:", type=("build", "run"))
```

Reply:
true, the code is python2 and python3. Are there any opinions about dropping python2 support?

Reply:
Python 2 support will be dropped in Spack after the next release, but it's probably better to document this in the package anyway.